scripts/bundleSizeConfig.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
-  MIN_SIZE: 547,
-  MAX_SIZE: 585,
+  MIN_SIZE: 552,
+  MAX_SIZE: 710,
```

Comment:
Debugging this, seems something in the later comments as caused the bundle size to increase which doesn't quite add up with the loadable work ...

Reply:
Founddd it, its caused by importing `@storybook/addon-knobs` in prod code https://github.com/bbc/simorgh/pull/4240#pullrequestreview-303465800

Reply:
LOLOLOLOLOLOLOLOL VisCode auto-complete/import ... 

![image](https://user-images.githubusercontent.com/7791726/67038962-57705700-f118-11e9-8ae3-c4455c6e21b5.png)